test/integration/db_migrations_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'test_helper'
+require 'lotus/environment'
+require 'lotus/commands/db'
+require  FIXTURES_ROOT.join('sql_adapter')
+
+
+describe 'CLI db migration' do
+  let(:app_name)       { nil }
+  let(:config)         { FIXTURES_ROOT.join('migrations/config/environment') }
+  let(:opts)           { Hash[environment: config] }
+  let(:adapter_config) { Lotus::Model::Config::Adapter.new(type: :sql, uri: SQLITE_CONNECTION_STRING) }
+  let(:sequel)         { Sequel.connect(adapter_config.uri) }
+  let(:env)            { Lotus::Environment.new(opts) }
+  let(:db_command)     { Lotus::Commands::DB.new(app_name, env) } 
+
+  before do
+    @current_dir = Dir.pwd
+    Dir.chdir FIXTURES_ROOT.join("migrations")
+    drop_tables
+  end
+
+  after do
+    Dir.chdir @current_dir
+    @current_dir = nil
+  end
+  
+  describe 'db migrate' do
+    before do
+      db_command.migrate   
+    end
+
+    describe 'when not pass lotus app' do
```

Comment:
What do you think about rewording this sentence to `when app param is not specified`?